extensions-core/multi-stage-query/src/test/java/org/apache/druid/msq/test/CalciteMSQTestsHelper.java

```diff
@@ -182,6 +190,21 @@ public String getFormatString()
     );
   }
 
+  private static LoadedSegmentDataProviderFactory getTestLoadedSegmentDataProviderFactory()
+  {
+    LoadedSegmentDataProviderFactory mockFactory = Mockito.mock(LoadedSegmentDataProviderFactory.class);
+    LoadedSegmentDataProvider loadedSegmentDataProvider = Mockito.mock(LoadedSegmentDataProvider.class);
+    try {
```

Comment:
Can you please what should developer do when they this this. Like a like to an example test case.

Reply:
Added